examples/classifying_shapes.ipynb

```diff
@@ -59,7 +59,8 @@
    "outputs": [],
```

Comment:
<p>do we still need to specify the <code>max_edge_length</code>?  if not, i would remove it for simplicity</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I guess it was put there to make the computation faster, I'm not sure!</p>

Reply:
<p>I believe this is indeed the case, computation would be slower without it.</p>